src/components/Comments.jsx

```diff
@@ -19,13 +23,66 @@ const Comments = ({ article_id, comments, setComments }) => {
       });
   }, [article_id, setComments]);
 
+  const handleCommentSubmit = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
```

Comment:
This loading state is shared with fetchCommentsByArticleID which means the whole page displays a loading screen when posting a comment. I would separate these out for a better user experience